src/GraphQL/Types/InputObjectGraphType.cs

```diff
@@ -12,15 +10,6 @@ public class InputObjectGraphType : InputObjectGraphType<object>
 
     public class InputObjectGraphType<TSourceType> : ComplexGraphType<TSourceType>, IInputObjectGraphType
     {
-        public override FieldType AddField(FieldType fieldType)
```

Comment:
Since it was necessary to simultaneously check for input and output types, it was more convenient to transfer them to the base class.

Reply:
In addition, this check did not take into account all cases.